frontend/src/api/Problem.ts

```diff
@@ -170,5 +170,6 @@ export const getDefaultCodeMap = (problemId: string): Promise<DefaultCodeType> =
   .get<DefaultCodeType>(routes.defaultCodeMap(problemId))
   .then((res) => res.data)
   .catch((err) => {
+    console.log("this is where it's all coming from");
```

Comment:
Are you going to keep this in on the merge? Do you want me to comment on this?

Reply:
No, sorry, this is one of the console.log statements I didn't see and delete before pushing this branch.